nifi-registry-core/nifi-registry-web-ui/src/main/webapp/nf-registry.js

```diff
@@ -80,23 +80,23 @@ NfRegistry.prototype = {
 };
 
 NfRegistry.annotations = [
-    new ngCore.Component({
+    new Component({
         selector: 'nf-registry-app',
-        template: require('./nf-registry.html!text'),
+        template: template,
```

Comment:
Little bit nit-picking, feel free to ignore... but you can write these in a shorter syntax:

```
{
	...
	template,
	...
}
```